src/_common/__tests__/Collapsible-test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import expect from 'expect';
+import {createRenderer} from 'react-addons-test-utils';
+import Collapsible  from '../Collapsible';
+
+describe('Collapsible', ()=>{
+	it('works', ()=>{
```

Comment:
more description, it would help new contributor understand what the component does